src/notifications/screens/notifications.screen.js

```diff
@@ -110,6 +116,16 @@ const styles = StyleSheet.create({
     textAlign: 'center',
     ...fonts.fontPrimary,
   },
+  markAllAsReadButton: {
+    marginTop: 3,
+    paddingTop: 3,
+    paddingBottom: 3,
+    marginLeft: 11,
+    marginRight: 11,
+    borderColor: colors.mercury,
+    borderWidth: 1,
+    borderRadius: 3,
+  },
 });
```

Comment:
I'm not a fan of this button design, and it looks too different from the "Merge pull request" button we have : 
![screen shot 2017-09-27 at 6 52 55 pm](https://user-images.githubusercontent.com/304450/30929026-87980fcc-a3bd-11e7-99a4-883b5603c3cd.png)

Reply:
Hmm this is interesting... I agree that we should keep to a similar style in general, but I don't think a bulky button like this taking up a bunch of space is the best UI. I like how this style is more discrete at the top of the screen.

Reply:
Absolutely! Right after commenting on this, I started implementing a standard <Button /> component to be used everywhere (small/normal/big, info/danger/..) :)

Reply:
Maybe yes, we can not make this button fixed, then it will be before the first notification, how do you do?

In style, I think this is the best option, I mean that the button is stretched to full width.